gci/management/commands/fetch_sub_linter_issues.py

```diff
@@ -0,0 +1,86 @@
+import argparse
+from os import environ
+
+from django.core.management.base import BaseCommand
+from termcolor import colored
+
+from gci.linter_repos import (
+  fetch_issues,
+  get_github,
+  get_repo_host,
+  get_repo_name,
+)
+
+
+class Command(BaseCommand):
+    help = 'Fetch issues from linter repositories and filter'
+
+    DEFAULT_LABELS = [
+        'easy',
+        'easy fix',
+        'first issue',
+        'good first issue',
+        'newcomer',
+        'help wanted',
+        'beginner',
+        'up-for-grabs',
+        'beginner-friendly',
+        'low-hanging fruit',
+        'hacktoberfest'
+    ]
+
+    DEFAULT_KEYWORDS = [
+        'easy fix',
+        'documentation',
+        'README',
+        'pep8',
+        'simple'
+    ]
+
+    def add_arguments(self, parser):
+        parser.add_argument('--labels', '-l', nargs='*')
+        parser.add_argument('--keywords', '-k', nargs='*')
+        parser.add_argument('--threads', '-t', type=int)
+        parser.add_argument('--all', action='store_true')
+        parser.add_argument(
+          'targets', type=argparse.FileType(),
+          help='file containing tagret repositories')
+
+    def handle(self, *args, **options):
+        hosts = {
+          'github': get_github(environ['GITHUB_TOKEN']),
+        }
+
+        with options.get('targets') as targets_file:
+            targets = targets_file.read().splitlines()
+
+        labels = options.get('labels')
+        if labels is None:
+            labels = Command.DEFAULT_LABELS
+
+        keywords = options.get('keywords')
```

Comment:
`keywords = options.get('keywords', self.DEFAULT_KEYWORDS)`

Reply:
Wouldn't work because `options.keywords` is set to `None` if no labels are passed.